README.md

```diff
@@ -1,54 +1,28 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/xqnglxgaalh442cj/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/socket_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Socket_Toolkit/Socket_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=92&branchName=master)
```

Comment:
Azure badge should be `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Socket_Toolkit/Socket_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=90&branchName=master)`